src/doc/rustc/src/jobserver.md

```diff
@@ -0,0 +1,55 @@
+# Jobserver
+
+Internally, `rustc` may take advantage of parallelism. `rustc` will coordinate
+with the build system calling it if a [GNU Make jobserver] is passed in the
+`MAKEFLAGS` environment variable.
```

Comment:
It may use other env vars too, like `CARGO_MAKEFLAGS`, not sure whether we should document the exhaustive list here.
If something goes wrong, then rust will print the specific env var that was used.

Reply:
I guess we could mention that other flags could also have an effect and/or link to the Cargo docs, since they are publicly documented anyway. Though it wouldn't hurt to document all env vars in `rustc` somewhere -- as a user, I would appreciate it, even if they are documented as "do not use".

I have added the mention in v4.